src/config.ts

```diff
@@ -0,0 +1 @@
+export const connectionString = "mongodb://localhost/loyer-paris-be"
```

Comment:
tu as changé avec la vraie valeur ?

Reply:
https://github.com/aymericdo/loyer-paris-be/pull/28/files#r372036629